tests/RA_UnitTestHelpers.h

```diff
@@ -17,33 +15,34 @@ namespace VisualStudio {
 namespace CppUnitTestFramework {
 
 // converters for asserting enum values
-
-template<> static std::wstring ToString<MemSize>(const MemSize& t)
+#pragma warning(push)
+#pragma warning(disable : 4505) // unreferenced inline functions
+template<> std::wstring ToString<MemSize>(const MemSize& t)
```

Comment:
`static template` overloads are undefined, unless the compiler adjusts for it. I've never seen static template overloads before either, it doesn't make any sense.

Anyway these functions were considered unreferenced inline functions but didn't want to deal with that now.